mock/py/mockbuild/util.py

```diff
@@ -617,20 +617,24 @@ def __init__(self, personality, chrootPath, cwd, uid, gid, env=None,
         getLog().debug("child environment: %s", env)
 
     def __call__(self, *args, **kargs):
-        if not self.shell:
-            os.setsid()
-        os.umask(0o02)
-        condUnshareNet(self.unshare_net)
-        condPersonality(self.personality)
-        condEnvironment(self.env)
-        # Even if nspawn is allowed to be used, it won't be used unless there
-        # is a chrootPath set
-        if not USE_NSPAWN or not self.chrootPath:
-            condChroot(self.chrootPath)
-            condDropPrivs(self.uid, self.gid)
-            condChdir(self.cwd)
-        condUnshareIPC(self.unshare_ipc)
-        reset_sigpipe()
+        try:
+            if not self.shell:
+                os.setsid()
+            os.umask(0o02)
+            condUnshareNet(self.unshare_net)
+            condPersonality(self.personality)
+            condEnvironment(self.env)
+            # Even if nspawn is allowed to be used, it won't be used unless there
+            # is a chrootPath set
+            if not USE_NSPAWN or not self.chrootPath:
+                condChroot(self.chrootPath)
+                condDropPrivs(self.uid, self.gid)
+                condChdir(self.cwd)
+            condUnshareIPC(self.unshare_ipc)
+            reset_sigpipe()
+        except:  # pylint: disable=bare-except
```

Comment:
```suggestion
        except BaseException:
```

Reply:
Thanks, should I note this as prefered way to catch **any** exception?
FWIW, tihs ends up with a slightly different PyLint error:
`broad-except: Catching too general exception BaseException`

Reply:
Yes, this is the preferred way. However since pylint seem to dislike it anyway, I have no idea what's the preferred way for pylint.